flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  description =
+    "Tool to simplify secure shell connections over AWS Systems Manager (SSM).";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs?ref=nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+        awssh = pkgs.python310Packages.buildPythonPackage rec {
```

Comment:
Is Python 3.11 not available?

Reply:
IIRC this package only works on 3.10 and breaks when I tried 3.11. There were too many issues.